src/datasources/redis/pmseries_srv.ts

```diff
@@ -172,6 +201,14 @@ export class PmSeriesSrv {
         return _.pick(this.labelCache, series);
     }
 
+    async getLabelsNames(): Promise<string[]> {
+        return await this.pmSeriesApi.labelsNames();
+    }
+
+    async getLabelValues(names: string[]): Promise<LabelsValuesResponse> {
```

Comment:
should be `getLabelsValues()` to match `this.pmSeriesApi.labelsValues()`

Reply:
I ll make both `get<singular><plural>`